addons/medical_treatment/functions/fnc_bandage.sqf

```diff
@@ -18,8 +19,10 @@
  * Public: No
  */
 
-params ["_medic", "_patient", "_bodyPart", "_classname"];
+[QGVAR(bandaged), _this] call CBA_fnc_localEvent; // Raise event with reference so mods can modify this
```

Comment:
```suggestion
_this set [4, _this param [4, 1]]; // set default Bandage effectiveness coefficient
[QGVAR(bandaged), _this] call CBA_fnc_localEvent; // Raise event with reference so mods can modify this
```

not sure if needed, 
but this might make it a little easier for different mods to work together if we start with the 1 in the array
e.g. events written like this wouldn't conflict
```
[QGVAR(bandaged), {
    params ["_medic", "", "", "", "_bandageEffectiveness"];
    if (_medic is special) {
        _this set [4, _bandageEffectiveness * 5];
```

Reply:
Good catch.